specification/resources/resource-manager/Microsoft.Resources/preview/2024-10-01-preview/deleteOptions.json

```diff
@@ -0,0 +1,303 @@
+            "$ref": "./examples/GetDeleteOptions.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK - Returns the delete options response definition.",
+            "schema": {
+              "$ref": "#/definitions/DeleteOptionsDefinition"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DeleteOptions"
+        ],
+        "operationId": "DeleteOptions_Delete",
+        "description": "Delete delete options of resource.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceProviderNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "Delete DeleteOptions": {
+            "$ref": "./examples/DeleteDeleteOptions.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK - Delete Options deleted."
+          },
+          "204": {
+            "description": "Delete Options does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DeleteOptionsDefinition": {
+      "description": "A delete options response definition.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DeleteOptionsProperties"
+        }
+      }
+    },
+    "DeleteOptionsProperties": {
+      "type": "object",
+      "description": "Delete option properties",
+      "properties": {
+        "deleteOptionsId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique GUID representing delete options."
+        },
+        "subscriptionId": {
```

Comment:
>subscriptionId [](http://example.com/codeflow?start=9&length=14)

if the deleteoptions are at rg level why is the subid and rg added in the response body?